services/src/main/java/org/keycloak/services/resources/admin/GroupsResource.java

```diff
@@ -118,14 +140,20 @@ public GroupResource getGroupById(@PathParam("id") String id) {
     @Produces(MediaType.APPLICATION_JSON)
     public Map<String, Long> getGroupCount(@QueryParam("search") String search,
```

Comment:
I think we should not change this method, at least for now. But only the one that actually does the query.

Reply:
Thanks. I've updated the count method similarily to the other method.